csaf_2.1/prose/edit/src/schema-elements-02-props-04-vulnerabilities.md

```diff
@@ -382,20 +382,29 @@ List of involvements (`involvements`) of value type `array` with 1 or more uniqu
 ```
 
 Every Involvement item of value type `object` with the 2 mandatory properties Party (`party`), Status (`status`) and
-the 2 optional properties Date of involvement (`date`) and Summary (`summary`) is a container that allows the document producers to
-comment on the level of involvement (or engagement) of themselves (or third parties) in the vulnerability identification, scoping,
-and remediation process.
+the 5 optional properties Party contact information (`contact`), Date of involvement (`date`), Group IDs (`group_ids`),
```

Comment:
I do not remember how we ended up using this extensive Upper Case Usage when followed already by a specific and controlled `term`, but it might enlarge the attention span of the readers when we use normal english case and expand ArtWords like IDs to identifiers if and only if being followed by a parenthiszed controlled term.

Finally-feeling-like-marketing-department-changing-wording-in-the-middle-of-trhe release-processly-yours,
Stefan.

Reply:
Those are the exact values of the `title` field from the JSON. If we want to change that, we need to be consistent and do it throughout the standard. Given your release plan and the list of tickets that should still go in there, I'm reluctant to tackle the problem now.
We could have that as separate issue - I guess it would only be a non-material change...